docs/src/tutorials/tutorial_4.md

```diff
@@ -0,0 +1,304 @@
+# **Tutorial 4:** Do you know our Mascot? - Learn about transitions and subactions!
+
+You have learned a couple of cool features of Javis already. Now you're ready to finally meet our little mascot. Well actually you can't just see him, we have to create him first. :smile:
+
```

Comment:
Could you add a section here about learning outcomes? Meaning, what a user will learn by following this tutorial?